crypto/keytools.go

```diff
@@ -11,3 +14,23 @@ const (
 	CurveEd25519   string = "ed25519"
 	CurveSecp256K1 string = "secp256k1"
 )
+
+func ObtainP2PKeypair(pubKey, privKey string) (priv crypto.PrivKey, pub crypto.PubKey, err error) {
```

Comment:
Obtain because Get is not allowed in Go? ;-) Why not call it `Load`? A bit more common.

Reply:
there is a minor difference between a `getter`(take a value that is already a field, no external calls to db etc), `obtainer`(load a value and execute some logic to derive a value) and a `loader`(load from some where) ;) , So I will keep it this way.